platform-core-module/src/main/java/com/softicar/platform/core/module/CoreI18n.java

```diff
@@ -767,6 +767,10 @@
 		.de("Reply-To");
 	I18n0 RESET_PASSWORD = new I18n0("Reset Password")//
 		.de("Passwort zurücksetzen");
+	I18n0 RETENTION_DAYS_OF_EXECUTIONS = new I18n0("Retention Days of Executions")//
+		.de("Speicherungstage für Ausführungen");
+	I18n0 RETENTION_DAYS_OF_EXECUTIONS_MUST_NOT_BE_SMALLER_THAN_0 = new I18n0("Retention days of executions must not be smaller than 0.")//
```

Comment:
I suggest to reword this, in order to get rid of the negation, e.g.:
`Retention days of executions must be at least 0.`
It's easier to grasp this way.

Reply:
done